core/src/stream.rs

```diff
@@ -103,8 +103,7 @@ impl<M: DecodeVersioned> Stream<M> for warp::ws::WebSocket {
     type Message = warp::ws::Message;
 }
 
-#[cfg(test)]
-mod tests {
+mod ws_client {
```

Comment:
why would you remove `[cfg(test)]`?

Reply:
Temporary workaround. I'm planning on introducing a feature flag for cross-crate testing.